d/d.bzl

```diff
@@ -47,6 +47,14 @@ def _create_setup_cmd(lib, deps_dir):
       deps_dir + "/" + lib.basename + "\n"
   )
 
+def _files_directory(files):
+  """Returns the shortest parent directory of a list of files."""
+  dir = files[0].dirname
+  for f in files:
+    if len(dir) > len(f.dirname):
```

Comment:
In this case it does not matter. I don't think we should fail either because we cannot really detect it.

We could do a real common prefix computation but that would be a bit more complex that a loop over the list of files (needs to find the largest common prefix between dir and the new file).